packages/stark-core/src/validation/validators/map-not-empty/map-not-empty.validator.fn.ts

```diff
@@ -0,0 +1,5 @@
+export const starkMapNotEmptyValidatorName: string = "starkMapNotEmpty";
+
+export function starkMapNotEmpty(map: Map<any, any>): boolean {
```

Comment:
Can you add a unit test for this validator? I know this is a really simple validator but in any case we should test it ;)

You can take the other tests as a basis